### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by addod19**

### CODE REVIEW

File: assets/stylesheet/style.css
Comment (bolah2009) [inline assets/stylesheet/style.css]: #### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: assets/stylesheet/style.css
Comment (bolah2009) [inline assets/stylesheet/style.css]: #### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: assets/stylesheet/content.css
Comment (bolah2009) [inline assets/stylesheet/content.css]: #### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: .stickler.yml
Comment (bolah2009) [inline .stickler.yml:2]: #### `REQUIRED`

Add appropriate `.stickler.yml` file to the root of the project. Let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml) instead. :point_down: 

``` suggestion
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job @addod19  :clap: Almost there :smile: 

### I see you have made most suggested changes in #1

The [clone](https://raw.githack.com/addod19/using-bootstrap/dev/index.html) looks nice, responsive and uses bootstrap class.

Before giving a :+1: let's make some **required** changes.
- Fix navbar for the large screen. Compare with source page.
- Add appropriate linter file. [See comment below](https://github.com/addod19/using-bootstrap/pull/2#discussion_r316188188)
   > [Set up a code linter](https://github.com/microverseinc/linters-config/tree/master/css).
   > Follow the [Github Flow guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).
- [OPTIONAL] Use more bootstrap classes
- Use semantic elements. Think of where semantic elements would have made the page more accessible. You can inspect [source page](https://www.newsweek.com/) for hints and see link below.
   > Remember to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).
- Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.
   > Follow the [Github Flow guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).
- [OPTIONAL] Use more bootstrap classes

Go through the [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/) and [odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap) project requirements for more details.


Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Comment by samgaco** [comment]
### Hi @addod19 :hand: 

#### You are putting a lot of effort and that is good, the project is looking lit :fire:  we just need some minor improvements before we can give you the approval:

* [REQUIRED] Read about [HTML semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp) and implement some of them (consider using `<main>` for the dominant content of the body, `<section>`and `<article>` tags, etc.)

* [REQUIERED] You have to use[ this linters file.](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml)

* [OPTIONAL] Make the navigation bar and the header looking closer to those in the original page.

* [OPTIONAL] Fix this problem of balance in the top story section.

> fix

### That's it. Please :pray: work on these changes and submit back for another code review once you are done. Regards!! :muscle:

**Comment by addod19** [comment]
i have created a .stickler.yml file.....How do i use linters? Can you be more specific. I have a stickler.yml file and how do i use it as specified by reviewer

**Comment by addod19** [comment]
I have followed the link and created stylelint.config.js file......

**Comment by Ispirett** [comment]
# Reviewing

**Review by Ispirett — APPROVED** [review]
Hey @addod19   Great job with all the changes made 👍 
image. 

This project looks good to go 👍 you can merge.

**[Optional] not required.** Not required to complete this project.
- [ ] In the **More Stories, Latest News, sections**, you can add some space between the images and the text.
- [ ] In the **Options sectio**n, the articles on the right, the text should be at the right side of the images.
## Git/Githubflow reminder.

Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

**Comment by addod19** [comment]
Alright.....Thank you

**Pull request merged**
